src/index.css

```diff
@@ -1,4 +1,338 @@
+@import url("https://fonts.googleapis.com/css?family=Montserrat");
+@import url("https://fonts.googleapis.com/css?family=Roboto Slab'");
+
 body {
   margin: 0;
   padding: 0;
+  background-color: #e7e3e3;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.bookStore {
+  width: 90vw;
+  height: 90vh;
+  margin: 3vh 5vw;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: whitesmoke;
+}
+
+.active {
+  color: black;
+  font-weight: 700;
+}
+
+.navBar {
+  display: flex;
+  flex-direction: row;
+  padding-left: 5rem;
+  padding-bottom: 1.5rem;
+  background-color: whitesmoke;
+}
+
+.links {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 1rem;
+  width: 15vw;
+}
+
+.book-container {
+  display: flex;
+  flex-direction: row;
+  width: 60vw;
+  margin: 1.313rem 3.25rem 0;
+  padding: 0.8rem 9.188rem 1.625rem 1.688rem;
+  padding-bottom: 2rem;
+  border-radius: 4px;
+  border: solid 1px var(--white-two);
+  background-color: white;
+  margin-left: 10%;
+}
+
+.inputForm {
+  width: 72vw;
+  height: 4.813rem;
+  margin: 1.313rem 4.85rem 0;
+  padding: 0;
+  padding-bottom: 3rem;
+  border-top: solid 1px #b4b0b0;
+  background-color: #e7e3e3;
+  margin-left: 13%;
+}
+
+.Title {
+  width: 10.875rem;
+  height: 1.5rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.Lesson-Pane2 {
+  width: 15.813rem;
+  height: 2.813rem;
+  border-radius: 4px;
+  border: solid 1px #888;
+  background-color: #fff;
+  margin-right: 2.2rem;
+  padding-left: 0.5rem;
+}
+
+.shadow {
+  width: 85.063rem;
+  height: 63.313rem;
+  opacity: 0.2;
+  border-radius: 4px;
+  -webkit-filter: blur(50px);
+  filter: blur(50px);
+  background-color: var(--black);
+}
+
+.titling {
+  width: 30rem;
+}
+
+.submit {
+  width: 10.5rem;
+  height: 2.813rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  color: white;
+}
+
+.remove {
+  width: 4.375rem;
+  height: 1.188rem;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: none;
+  cursor: pointer;
+  border-right: solid 1px #d6d3d3;
+  background-color: white;
+}
+
+.comment {
+  padding-right: 3rem;
+  width: 5rem;
+}
+
+.edit {
+  padding-left: 1rem;
+  width: 3rem;
+  border: none;
+  cursor: pointer;
+  background-color: white;
+}
```

Comment:
- [ ] Why don't you make this `Edit` have the same color as the `Delete` and `Comment` so as to match the design guideline and also as suggested by the previous reviewer?😄

Your Implementation | Zeplin Design
:---:|:---:
![](https://i.imgur.com/Oq0LgqH.png) | ![](https://i.imgur.com/ySvzM7P.png)